src/software/world/ball_state_test.cpp

```diff
@@ -57,3 +59,14 @@ TEST(BallStateTest, compare_states_with_different_distances_from_the_ground)
     EXPECT_FALSE(ball_state_1 == ball_state_2);
     EXPECT_TRUE(ball_state_1 != ball_state_2);
 }
+
+TEST(BallStateTest, construct_with_protobuf)
+{
+    BallState ball_state_1(Point(1, -2.3), Vector(0, 0.4), 0.1);
+    Ball ball(ball_state_1, Timestamp());
+    std::unique_ptr<TbotsProto::BallState> ball_state_proto = createBallState(ball);
+    BallState ball_state_2(*ball_state_proto);
+
+    EXPECT_FALSE(ball_state_1 == ball_state_2);
+    EXPECT_TRUE(ball_state_1 != ball_state_2);
```

Comment:
these two expect statements look like they're testing exactly the same thing no?

Reply:
You're right. I removed one of the cases